src/ReadLog.jl

```diff
@@ -0,0 +1,134 @@
+# Read LCM log files directly
+
+# export
+#   LCMlog,
+#   close,
+#   readNextEvent,
+#   handle,
+#   subscribe
+
+# also see JuliaLang's RawFD type, but little information found.
+# www.c4learn/c-programming/c-file-structure-and-file-pointer
+mutable struct FILE
+  level::Cshort
+  token::Cshort
+  bsize::Cshort
+  fd::Cuchar
+  flags::Cuint  # (unsigned flags ;)
+  hold::Cuchar
+  buffer::Ptr{Cuchar}
+  curp::Ptr{Cuchar}
+  istemp::Cuint # (unsigned is temp)
+end
+
+mutable struct lcm_eventlog_t
+  f::Ptr{FILE}
+  eventcount::Clonglong
```

Comment:
In `eventlog.h` this is explicitly an `int64_t`, so we should use an `Int64` here as well.

Reply:
Okay, I was under the impression I had to use the C types. I followed the `ccall` [reference for wrapping structs](https://docs.julialang.org/en/stable/manual/calling-c-and-fortran-code/#Mapping-C-Types-to-Julia-1). You suggesting we just go with?
```julia
mutable struct lcm_eventlog_t
  f::Ptr{Void}
  eventcount::Int64
end
```

Reply:
Yeah, exactly. You're right that using the C types is generally the right thing to do, but there's nothing special about them. For example, on my system, `Cint === Int32`. The reason they're recommended is that if you have a C library that uses an `int`, then you want Julia to use the same type, regardless of what `int` means in your particular architecture. But the LCM C library explicitly uses `int64_t` here, so the right way to ensure Julia is using the same type is also to use Int64.